src/EditorFeatures/Test/Diagnostics/DiagnosticTagSourceTests.cs

```diff
@@ -60,34 +68,17 @@ public void Test_TagSourceDiffer()
 
                 service.WaitUntilCompletion_ForTestingPurposesOnly(workspace, incrementalAnalyzers);
 
-                diagnosticWaiter.CreateWaitTask().PumpingWait();
-                squiggleWaiter.CreateWaitTask().PumpingWait();
+                listener.CreateWaitTask().PumpingWait();
 
-                taggerSource.TestOnly_Dispose();
+                snapshot = workspace.Documents.First().GetTextBuffer().CurrentSnapshot;
+                spans = tagger.GetTags(new NormalizedSnapshotSpanCollection(new SnapshotSpan(snapshot, 0, snapshot.Length))).ToList();
+                Assert.True(spans.First().Span.Contains(new Span(0, 1)));
 
+                ((IDisposable)tagger).Dispose();
```

Comment:
Can't.  Taggers don't extend from IDisposable.  So 'using' complains.  And if i type it as IDisposable, then i can't call ProduceTagsAsync on it.  I could type it as IDisposable, and then cast the result to the ITagger type... but this seems like it's not worth it.